src/Datadog.Trace/Agent/Api.cs

```diff
@@ -35,19 +34,7 @@ public Api(Uri baseEndpoint, IApiRequestFactory apiRequestFactory, IDogStatsd st
             _apiRequestFactory = apiRequestFactory ?? CreateRequestFactory();
 
             // report runtime details
-            try
-            {
-                _frameworkDescription = FrameworkDescription.Create();
-
-                if (_frameworkDescription != null)
-                {
-                    Log.Information(_frameworkDescription.ToString());
-                }
-            }
-            catch (Exception e)
-            {
-                Log.SafeLogError(e, "Error getting framework description");
-            }
+            Log.Information(FrameworkDescription.Instance.ToString());
```

Comment:
Maybe we can remove this log line completely? It was here because historically it was the first place we ever tried to get this information (to add it to the http headers to the agent). Now we include this in the [startup log](https://github.com/DataDog/dd-trace-dotnet/blob/0539739a8da8a3825c55481baa63247594013531/src/Datadog.Trace/Tracer.cs#L502) using a consistent format across language tracers.

Reply:
Removed